doc/source/whatsnew/v0.19.0.txt

```diff
@@ -40,6 +40,8 @@ Other enhancements
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+- ``DataFrame.to_csv()`` has dropped the ``engine`` parameter (:issue:`13419`)
```

Comment:
Can you move this to "Removal of prior version deprecations/changes" (a bit below)